src/test/resources/org/jabref/logic/importer/fileformat/BibTeXMLImporterTestArticle2.bib

```diff
@@ -1,9 +1,9 @@
-@article{Mustermann2016,

```

Comment:
Is there any reason why you changed the formatting of the file?
if not please revert